topics/climate/tutorials/fates/tutorial.md

```diff
@@ -53,34 +58,34 @@ It will also teach you on how to create Galaxy workflow for your CLM-FATES simul
 
 > ### {% icon comment %} Background
 >
-> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”.
+> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”, which is a vegetation demographic model ([Fisher et al. 2018](https://onlinelibrary.wiley.com/doi/full/10.1111/gcb.13910)).
 > FATES needs what we call a "Host Land Model" (HLM) to run and in this tutorial
 > we will be using the [Community Land Model](http://www.cesm.ucar.edu/models/clm/)
 > of the [Community Terrestrial Systems Model](https://github.com/ESCOMP/CTSM) (CLM-CTSM).
 > FATES was derived from the CLM Ecosystem Demography model (CLM(ED)), which was documented in
-> {% cite Fisher2015 %}.
+> {% cite Fisher2015 %} and [Koven et al. 2020](https://bg.copernicus.org/articles/17/3017/2020/). 
```

Comment:
could you add this citation to the `tutorial.bib` file? Then you can use the same syntax as the paper before it `{% cite X %}`

You can use https://www.doi2bib.org/ to convert a doi into a bibtex entry to put in this file